docs/docs/guides/slo.md

```diff
@@ -149,8 +149,8 @@ so, in this case, the query becomes:
 
 The other key-value pairs such as 'project' and 'stage' are just examples of how one could pass to the provider
 information similar to Keptn v1 objectives.
-For a working example you can
-check [here](https://github.com/keptn/lifecycle-toolkit/tree/main/test/testanalysis/analysis-controller-multiple-providers).
+For a working example, you can
+check [here](../../../test/kuttl/testanalysis/analysis-controller-multiple-providers).
```

Comment:
```suggestion
check [here](https://github.com/keptn/lifecycle-toolkit/tree/main/test/kuttl/testanalysis/analysis-controller-multiple-providers).
```

I will eventually migrate this under an asset file in the docs

Reply:
Reverted for now, will fix in a follow up PR

Reply:
this doesn't need to be fixed anyways. relative links in the docs should only be used for other things that are inside the docs (i.e. other docs pages or code snippets under the `docs` folder)